app/pom.xml

```diff
@@ -752,6 +752,18 @@
         <version>2.5</version>
       </dependency>
 
+      <dependency>
+        <groupId>org.jolokia</groupId>
+        <artifactId>jolokia-client-java</artifactId>
+        <version>1.3.7</version>
```

Comment:
1.4.0 is the latest version currently (though there is  not much change to 1.3.7)